Library/Homebrew/sandbox.rb

```diff
@@ -8,11 +8,12 @@ class Sandbox
   ].freeze
 
   def self.available?
-    OS.mac? && File.executable?(SANDBOX_EXEC)
+    OS.mac? && OS::Mac.version >= "10.6" && File.executable?(SANDBOX_EXEC)
```

Comment:
You can see commit message for explanation. Basically it exists but too old to support DSL we use.